praetorian_cli/ui/console/commands/marcus.py

```
@@ -499,5 +499,99 @@ def _cmd_research(self, args):
+        if subcmd == 'start':
+            if not rest:
+                self.console.print('[error]Usage: hunt start "<prompt>" [--duration 24h] [--scope #asset#...][/error]')
+                return
+            prompt = ' '.join(rest)
+            from datetime import datetime, timezone, timedelta
+            expires_at = (datetime.now(timezone.utc) + timedelta(hours=24)).strftime('%Y-%m-%dT%H:%M:%SZ')
+            with self.console.status('Launching hunt...', spinner='dots', spinner_style=self.colors['primary']):
+                result = self.sdk.post('hunt', {'prompt': prompt, 'expiresAt': expires_at})
+            uuid = result.get('uuid', '?')
+            status = result.get('status', '?')
+            self.console.print(f'[bold red]Hannibal is at the gates![/bold red]')
```

Ruff (0.15.20)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)